openid-4-verifiable-presentations-1_0.md

```diff
@@ -232,7 +232,14 @@ Presentation of Verifiable Credentials using OpenID for Verifiable Presentations
 
 The Authorization Request follows the definition given in [@!RFC6749] taking into account the recommendations given in [@!I-D.ietf-oauth-security-topics].
 
-The Verifier may send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101].
+The Verifier MAY send an Authorization Request as a Request Object either by value or by reference, as defined in the JWT-Secured Authorization Request (JAR) [@RFC9101].
+
+The Wallet may want to provide to the Verifier details about its technical capabilities to
+allow the Verifier to generate a request that matches technical capabilities of that Wallet.
+To enable this, the first Authorization Request can contain `request_uri_method` parameter with the value `post`
+that signals to the Wallet that it can make an HTTP POST to the Verifier's `request_uri`
+endpoint with the information about its capabilities. The Wallet MAY continue with JAR
+when it receives `request_uri_method` parameter with the value `post` but does not support this new feature.
```

Comment:
just tried to minimally fix-up some grammar 

```suggestion
The Wallet may want to provide to the Verifier details about its technical capabilities to
allow the Verifier to generate a request that matches the technical capabilities of that Wallet.
To enable this, the first Authorization Request can contain a `request_uri_method` parameter with the value `post`
that signals to the Wallet that it can make an HTTP POST requst to the Verifier's `request_uri`
endpoint with information about its capabilities. The Wallet MAY continue with JAR
when it receives `request_uri_method` parameter with the value `post` but does not support this new feature.
```

Reply:
@bc-pi `requst` -> `request`

Reply:
> @bc-pi `requst` -> `request`

Thx, fixed it in the suggestion

Reply:
I would suggest omitting `new` in be `the new feature`.

Reply:
committed suggestion and removed "new" afterwards